website/src/blogs/storage-policies-its-different-this-time.md

```diff
@@ -0,0 +1,77 @@
+
+**Fact #4** — All these also mean the intents that were once the sole prerogative of humans can now be built and operated by machines as well. This too with the same ease that the humans used to enjoy. After all, the ingredients *(#1, #2 and #3)* to make this possibility are all in place. This has come of age and is really an advancement in my opinion.
+
+***Bringing your own YAML really means setting your own policies and having complete control over their execution as well.***
+
+### Use the tool(s) you always loved
+
+A policy alone cannot justify its existence unless it is complimented with simple tools and automated processes. The careful reader might have already guessed it. Yes, I am talking of aligning storage policies with DevOps to realise its full potential.
+
+*Making storage policies more visible, more obvious, and enabling them to the enterprise’s established processes will make them simple to be believed and instil the faith to operate*. *Once again, there is a learning involved but the curve is not steep*.
+
+These policies should offer finest levels of control to the operators’ tools, their bots and of-course operators themselves when such a need arises. *For example finer granularity is craved for during rollbacks, automated downgrades or blue green deployments which are not uncommon in the world of storage infrastructures.*
+
+To complete the DevOps cycle, these policies which can be hand crafted or system generated can be submitted to the approver(s) (*which again can be a combination of humans as well as their loved tools*) as Pull Requests before being installed and applied against the storage.
+
+### Storage was the proverbial “Missing Cog”
+
+We are seeing users finally achieving what many have dreamt of for so long — ***storage****(and the rest of the infrastructure)* truly being driven by the needs of the application and in a way that remains understandable and for a variety of reasons *(take infra as code for example)* trusted by humans. And now increasingly we see the recognition that containerised storage itself is another important ingredient.
+
+Thanks to [OpenEBS](http://openebs.io) and more broadly containerised storage. *For the first time every team and workload can have its ****own fully functional storage controller***, with capabilities that have always been required by enterprise storage systems and that are still useful in taking care of *stateful workloads*. Our users do not want to give up the tools they used for the care and feeding of MySQL for example just because it now runs in a container. *This is possible by enabling capabilities like snapshots, versioning, encryption and more ****as knobs**** to be able to be turned on/off for each workload.*
```

Comment:
```suggestion
Thanks to [OpenEBS](http://openebs.io) and more broadly containerised storage. *For the first time, every team and workload can have its **own fully functional storage controller**, with capabilities that have always been required by enterprise storage systems and that are still useful in taking care of *stateful workloads*. Our users do not want to give up the tools they used for the care and feeding of MySQL for example just because it now runs in a container. *This is possible by enabling capabilities like snapshots, versioning, encryption, and more **as knobs** to be able to be turned on/off for each workload.*
```